Moise1/Politico
# Polish the index page

#### What does this PR do?

* Validate  user input using JavaScript to see if all fields are filled as required
#### Description of Task to be completed?
##### _Have the following fields working for Sign Up:_
* Full name (required)
* Location(required)
* Birth date(required) 
* Email (required) 
* Password(required)

#### How should this be manually tested?
* Try to leave an input field unfilled and see the error message. 
* Type a password whose length is less than 6 characters and see the error message. 
#### Any background context you want to provide?
* None
#### What are the relevant pivotal tracker stories?
[##163897599](https://www.pivotaltracker.com/n/projects/2240032)
#### Screenshots (if appropriate)
#### Questions:
